index.bs

```diff
@@ -15,6 +15,10 @@ Markup Shorthands: css no, markdown yes
 Assume Explicit For: yes
 </pre>
 
+<pre class=link-defaults>
+spec:infra; type:dfn; text:user agent
```

Comment:
What happens without this?

Reply:
```
LINE ~146: Multiple possible 'user agent' dfn refs for '['/']'.
Arbitrarily chose https://infra.spec.whatwg.org/#user-agent
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:infra; type:dfn; text:user agent
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:css2; type:dfn; for:/; text:user agent
  https://www.w3.org/TR/CSS21/conform.html#ua
  https://www.w3.org/TR/CSS21/conform.html#user-agent
[=/user agent=]
```

Reply:
Sounds like CSS should be fixed, but okay.